status-im/status-react
# Token input refactoring

fixes #21017

### Summary
Token input refactored:
- removed unused code for custom selection support
- removed unused possibility to control value internally
- removed internal conversion logic
- no internal state left
- component in the bottom left corner made a property because token input will be used with another one in upcoming screens

Controlled input namespace improved:
- bignumbers everywhere
- conversion functions added

`Send` screen refactored, bunch of unnecessary properties removed.



##### Functional

- wallet / transactions


### Steps to test
- Open Status
- Open send screen
- enter amount to send
- swap between amounts
- check that there is no visible changes in component behaviour in comparison to previous versions


status: ready
